radiosity.cpp

```diff
@@ -61,8 +74,10 @@ float form_factor(
     const Tree& tree,
     const Tree::TriangleId from_id,
     const Tree::TriangleId to_id,
-    const size_t num_samples)
+    const size_t num_samples = 128)
```

Comment:
Hmm. Also bei foo(1) wird natürlich for(1, 0) ausgeführt. Ich denke default Parameters sind in c++ mithilfe von Funktionsueberladung implementiert. Im gewissen Sinne ist es also nur syntatic sugar.